utils/src/main/java/org/web3j/utils/Numeric.java

```diff
@@ -112,6 +112,14 @@ public static String toHexStringNoPrefix(byte[] input) {
     public static String toHexStringWithPrefixZeroPadded(BigInteger value, int size) {
         return toHexStringZeroPadded(value, size, true);
     }
+    
+    public static String toHexStringWithPrefixSafe(BigInteger value) {
```

Comment:
Please can you add a few test cases for this specific method? Appreciate that the RequestTest provides coverage, but I'd like something explicit.

Reply:
Yeah sure, I'll write it out by the end of the day